mtl/multitask_tutorial.py

```diff
@@ -3,12 +3,16 @@
 # # Multi-Task Learning (MTL) Basics Tutorial
 
 # %% [markdown]
-# Multi-task learning is becoming a standard tool for the modern ML practioner. A major requirement of this approach is the ability to easily *add new datasets, label sets, tasks, and metrics* (and just as easily remove them). Thus, in the Snorkel multi-task model design, each of these concepts have been decoupled.
+# Multi-task learning is becoming a standard tool for the modern ML practioner.
+# (See Sebastian Ruder's [survey](http://ruder.io/multi-task/) from 2017 for a nice overview).
+# It often leads to computational gains (one model performing many tasks takes up less memory and storage) as well as performance gains (learning to do well on a related _auxiliary_ task can improve the model's ability on the _primary_ task).
+# A major requirement of multi-task learning in general is the ability to easily *add new datasets, label sets, tasks, and metrics* (and just as easily remove them). Thus, in the Snorkel multi-task model design, each of these concepts have been decoupled.
```

Comment:
> in the Snorkel multi-task model design

the what? shouldn't we say Snorkel comes with a pytorch-based modeling framework that can do MTL first? and position it relative to other things? (not necessary to use but...)